cors.go

```diff
@@ -227,7 +227,7 @@ func (c *Cors) HandlerFunc(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
-// Negroni compatible interface
+// ServeHTTP: Negroni compatible interface
```

Comment:
To be more go idiomatic, prefer "ServeHTTP provides Negroni compatible interface."